torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -131,6 +131,46 @@ FunctionParameter::FunctionParameter(const std::string& fmt, bool keyword_only)
   }
 }
 
+PyObject* handle_torch_function(PythonArgs &r, PyObject* args, PyObject* kwargs, PyObject* torch_api, const char* module_name) {
```

Comment:
Note that the API of this function changed after I moved it to this file. I'm sorry that makes this a little difficult to review!

I'm now passing in `torch_api` as a `PyObject*`. I'm also passing in the module name to facilitate the code that generates the error message below. Both of these changes facilitate using this function from both `python_torch_functions.cpp` and `python_nn_functions.cpp`.

There should be no other changes to this function.